optaplanner-core/src/main/java/org/optaplanner/core/config/solver/SolverConfig.java

```diff
@@ -257,6 +293,37 @@ protected RandomFactory buildRandomFactory(EnvironmentMode environmentMode_) {
         return randomFactory;
     }
 
+    protected Integer resolveMoveThreadCount() {
```

Comment:
Could use unit tests

Reply:
Yes, let's make a jira around this? I 'll add some unit tests, but probably after summit (merging won't be immediate anyway).

Reply:
https://issues.jboss.org/browse/PLANNER-1099